apps/tailwind-components/components/input/Ref.vue

```diff
@@ -26,7 +26,7 @@ const props = withDefaults(
   >(),
   {
     isArray: true,
-    limit: 10,
+    limit: 5,
```

Comment:
(A comment for the future)

All input components that use the limit property should have the same value to make sure the UX is consistent across the forms.